### ISSUE TITLE
[BUG] xkcd doesn't embed alt text from json

### ISSUE THREAD

**Issue by MrMrRubic** [rHomelab/LabBot-Cogs#274]
**Describe the bug**
When the bot retrieves XKCD comics, it's supposed to also embed the alt text from the json if there is any, though this doesn't happen.

**To Reproduce**
1. make the bot retrieve an XKCD comic with alt text (for example [2966](https://xkcd.com/2966/))
2. Notice the lack of embedded alt text.

**Expected behavior**
Alt text should be embedded if it exists.

**Screenshots**
![xkcd_screenshot](https://github.com/user-attachments/assets/c38a7e1d-40c4-42b2-8b4e-58142e98685c)

**Additional context**
[JSON of the relevant comic](https://github.com/user-attachments/files/16469714/info.0.json)

**Comment by tigattack** [comment]
How would you suggest this be shown? Joined to the titled with a ` - ` separator? A third field titled "Description", shown when alt-text is present? In the embed footer? Something else?

**Comment by MrMrRubic** [comment]
Well, considering the existing code for it in xkcd.py: 
```
# If there is alt text add it to the embed, otherwise don't
        if not data["alt"]:
            xkcd_embed.add_field(name="Comic Alt Text", value=data["alt"])
        else:
            pass
```
Unless i'm reading it wrong (which is likely considering how little proficient i am with python), it seems backwards? It checks if there isn't data in `alt`, and if there isn't it makes the field. If there is data, it skips it. So it should be the other way? Skip if nothing there, othwise make the field. So maybe something like: 
```
# If there is alt text add it to the embed, otherwise don't
        if data["alt"]:
            xkcd_embed.add_field(name="Comic Alt Text", value=data["alt"])
        else:
            pass
```
or
```
# If there is alt text add it to the embed, otherwise don't
        if data["alt"]:
            pass
        else:
            xkcd_embed.add_field(name="Comic Alt Text", value=data["alt"])
```

**Comment by tigattack** [comment]
Ha, I didn't even notice there was existing code for it 🤦 You're spot on. That's an even easier fix than I imagined. Thanks for the report!
